APSHTTPClient/APSHTTPRequest.m

```diff
@@ -413,6 +413,8 @@ - (void)responseFinished
   [self invokeCallbackWithState:APSHTTPCallbackStateDataStream];
 
   [self invokeCallbackWithState:APSHTTPCallbackStateLoad];
+  
+  [[self postForm] destroyTemporaryData];
```

Comment:
This line should be added in  error block of 
"- (void)URLSession:(nonnull NSURLSession *)session task:(nonnull NSURLSessionTask *)task didCompleteWithError:(nullable NSError *)error".

Reply:
Mhh, it is already. But it should also be here to be used on successful POST requests. Or do you have objections there?

Reply:
This is fine here. But in following section also it should be there. Reason is function get return from error block, if error happens. 
` - (void)URLSession:(nonnull NSURLSession *)session task:(nonnull NSURLSessionTask *)task didCompleteWithError:(nullable NSError *)error
 {
   if (error != NULL) {
     DebugLog(@"%s", __PRETTY_FUNCTION__);
     self.response.readyState = APSHTTPResponseStateDone;
     [self invokeCallbackWithState:APSHTTPCallbackStateReadyState];
 
     self.response.connected = NO;
     self.response.error = error;
     [self invokeCallbackWithState:APSHTTPCallbackStateError];
     return;
   }`